src/video/SDL_sysvideo.h

```diff
@@ -417,6 +417,7 @@ struct SDL_VideoDevice
         int retained_backing;
         int egl_platform;
         int driver_loaded;
+        int swap_behavior_preserved_bit;
```

Comment:
The EGL data struct in SDL_egl.c is probably a better place for this, as it is EGL specific.

Reply:
Moved